src/style.css

```diff
@@ -0,0 +1,149 @@
+html {
+  font-family: Roboto, sans-serif;
+  font-size: 10px;
+  line-height: 14px;
+  color: #616070;
+}
+
+body {
+  margin: 0;
+  padding: 0;
+}
+
+* {
+  box-sizing: border-box;
+}
```

Comment:
Split your css code to different files - BEM blocks.

According to [checklist](https://github.com/mate-academy/layout_product-cards/blob/master/checklist.md)

`11.[FILE STRUCTURE] - You should create separate file for each BEM block`

detailed info how to do it in [this video](https://mate.academy/learn/css-advanced/bem#/video/917)